packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/repositories/{{#enable_feature_counter}}counter_repository.dart{{/enable_feature_counter}}

```diff
@@ -1,8 +1,8 @@
 {{> licence.dart }}
 
+import '../../feature_counter/models/count.dart';
```

Comment:
All files in the base folder should be completely decoupled from the features. With that being said I believe the model should not be moved to the feature but should reside in the base/models folder instead

Reply:
Done